package.json

```diff
@@ -21,26 +21,27 @@
     "url": "https://github.com/Sage/carbon/issues"
   },
   "homepage": "https://github.com/Sage/carbon#readme",
+  "peerDependencies": {
+    "flux": "^3.0.0",
```

Comment:
We kind of do, as running `npm install` will install the latest v3 Flux i.e. 3.1.2.

However, is it better to offer more flexibility and say Carbon will work with any v3 of Flux? Then other apps don't have to update Flux as often.